Install-DVWA.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# Get language prefix
+lang_prefix="${LANG:0:2}"
+
+# Function for verifying the language and displaying the corresponding message
+get_language_message() {
+    if [[ $lang_prefix -eq "es" ]]; then
```

Comment:
An interesting fix for the problem, I like it!

Reply:
Thank you! with that It will be easy to add support for new lenguages on a future if it needs.